kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -70,8 +69,195 @@
     </KGrid>
 
     <hr class="bottom-border">
+    <div v-if="isQuestionAvailable">
+      <KGrid>
+        <KGridItem
+          :layout12="{ span: 6 }"
+        >
+          <div class="left-column-alignment-style">
+            <div class="align-kcheckbox-style">
+              <p>
+                <KCheckbox />
+              </p>
+            </div>
+
+            <div>
+              <p>{{ $tr('selectAllLabel') }}</p>
+            </div>
+          </div>
+        </KGridItem>
+
+        <KGridItem
+          :layout12="{ span: 6 }"
+        >
+          <div class="right-alignment-style">
+            <KGrid>
+              <KGridItem :layout12="{ span: 4 }">
+                <div class="icon-container">
+                  <KIcon class="reduce-chervon-spacing" icon="chevronDown" />
+                  <KIcon class="reduce-chervon-spacing" icon="chevronUp" />
+                </div>
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+
+                <KIcon
+                  class="icon-size"
+                  icon="refresh"
+                />
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+                <KIcon
+                  class="icon-size"
+                  icon="trash"
+                />
+              </KGridItem>
+            </KGrid>
+          </div>
+        </KGridItem>
+
+      </KGrid>
+
+      <AccordionContainer
+        :items="placeholderList"
+      >
+        <template
+          #default="{ isItemExpanded, toggleItemState }"
+        >
+          <div
+            v-for="item in placeholderList"
+            :key="item.id"
+          >
+            <AccordionItem
+              :id="item.id"
+              :key="item.id"
+              :items="placeholderList"
+              :title="item.title"
+              :expanded="isItemExpanded(item.id)"
+            >
+              <template
+                #heading="{ title }"
+                :accordionToggle="onAccordionToggle(item.id)"
+              >
+                <a
+                  @click="toggleItemState(item.id)"
+                >
```

Comment:
I think this should probably be a `button` for HTML semantics - an `a` tag is only "valid" when it has an `href` attribute.

Additionally this will need to have the [`aria-expanded` ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded).

Also, I think that we'll want to have an `aria-label` here indicating that this is a toggle button, ie "Toggle visibility of {question title}" - the `aria-expanded` attribute should already tell the screen reader user the state of the expansion, then the button aria-label should describe what the button does I think. [For reference "Button Role"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role)

Definitely will appreciate @radinamatic 's thoughts on how we can best label this interaction.